test/integration/assistant.v2.test.js

```diff
@@ -16,6 +16,10 @@ describe('assistant v2 integration', function() {
       assistant_id,
     };
     assistant.createSession(params, function(err, res) {
+      if (err) {
```

Comment:
The following snippet seems to be repeated a lot in the code.
```js
if (err) {
  expect(err.code).toBe(200);
}
```

When you find yourself writing the same code more than 3 times then it's time to rethink the solution.

Maybe you can try something like the following:

```js 
const checkErrorCode = (code, cb) => (err, res) => {
  if (err) {
     expect(err.code).toBe(code);
  }
  return cb(err, res);
}
```
Then you can do:

```js
assistant.message(params, checkErrorCode(200, function(err, res) {
  // checks...
});
```